charts/ocis/values.yaml

```diff
@@ -1482,6 +1482,8 @@ services:
   # -- PROXY service.
   # @default -- see detailed service configuration options below
   proxy:
+    # -- Specifies additional policies for the proxy service.
```

Comment:
I didn't know that such a feature exists. But it comes in very handy. What about the precedence when it comes to polices vs addionalPolicies? Basically: which one wins? and which is the merge-key?

Reply:
https://github.com/owncloud/ocis/blob/ca65278546f98fb21ac65744024d32fd93685b96/services/proxy/pkg/config/defaults/defaultconfig.go#L333-L349

If I read the code correctly: if there's a policy with that name already the specified routes are appended otherwise a new policy with that name is inserted with the routes specified here. So as I said, I don't think there's any magic merging involved

Reply:
@dschmidt ok, I confused policies for routes actually. Additional routes within a policy are appended to the end and therefore should have less priority. 
But you should definitely add a commented policy example. Please also highlight that it's only gonna have an effect if the policy is called "ocis", since we don't have any switch exposed in the helm chart to pick another policy.

Reply:
Updated. I left it a bit vague, as I don't feel like adding documentation here that actually is missing for oCIS (and in the worst case saying something that is not correct)

Okay for you?